test/parallel/test-http2-write-finishes-after-stream-destroy.js

```diff
@@ -0,0 +1,62 @@
+// Flags: --expose-gc
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const assert = require('assert');
+const http2 = require('http2');
+const makeDuplexPair = require('../common/duplexpair');
+
+// Make sure the Http2Stream destructor works, since we don't clean the
+// stream up like we would otherwise do.
+process.on('exit', global.gc);
```

Comment:
I'm not sure I fully get why the explicit `gc` is needed in this test. Could you elaborate? Are you just testing for a crash?

Reply:
@apapirovski Yea, I had to make changes to my original patch for this so that this `gc()` call wouldn’t crash in the destructor.

So yes, it’s not needed, it’s just an extra test

Reply:
Ok, awesome. Just wanted to make sure I wasn't missing anything. Thank you for replying. :)